umbral-pre-wasm/pkg-template/README.md

```diff
@@ -0,0 +1,107 @@
+# JavaScript bindings for `umbral-pre`
```

Comment:
Is it the same `README` as the one in the root of `umbral-pre-wasm`? Can we copy it from there on `make`?

Reply:
And `LICENSE` as well.